torch/__init__.py

```diff
@@ -248,7 +248,7 @@ def __bool__(self):
     # Magic methods installed by torch.fx.experimental.symbolic_shapes
 
     def __sym_int__(self):
-        ...
+        raise NotImplementedError("NYI")
```

Comment:
we didn't install `__sym_int__` properly on `torch.SymFloat` earlier, so this was quietly returning `None`. Added to fail louder

Reply:
Let's raise AssertionError here, this code should always be dead

Reply:
or just delete these stubs